src/EthereumTransaction.js

```diff
@@ -171,7 +171,7 @@ export default class EthereumTransaction extends Transaction {
      * @param {FileId} callData
      * @returns {this}
      */
-    setCallData(callData) {
+    setCallDataFileId(callData) {
```

Comment:
Unfortunately we cannot just change the name here because it's already release. We need to create a new method and deprecate the old one.